src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -44,10 +40,12 @@ private static Assembly AssemblyResolve_ManagedProfilerDependencies(object sende
                 && assemblyName.FullName.IndexOf("PublicKeyToken=def86d061d0d2eeb", StringComparison.OrdinalIgnoreCase) >= 0
                 && File.Exists(path))
             {
+                StartupLogger.Log("Loading {0}", path);
```

Comment:
Can you add the actual call `Assembly.LoadFrom`? This LoadFrom is different than the later `DependencyLoadContext.LoadFromAssemblyPath` so it might be helpful to know when each is invoked.